docsets/Discord4J/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "Discord4J",
-    "version": "2.10.0",
+    "version": "2.10.1",
     "archive": "Discord4J.tgz",
```

Comment:
You need to update this archive as well, not just the specific_version one.

Reply:
Added!